cluster/centos/util.sh

```diff
@@ -47,15 +47,6 @@ function detect-masters() {
   echo "KUBE_MASTER_IP_ADDRESSES: [${KUBE_MASTER_IP_ADDRESSES[*]}]" 1>&2
 }
 
-# Get node IP addresses and store in KUBE_NODE_IP_ADDRESSES[]
```

Comment:
Why deleting this function?

Reply:
The first commit in this PR removes some code that I'm pretty sure is unused. I came across it while making my other changes. Would it be worth fixing this upstream as part of my PR, or as a separate one?